src/v0/destinations/google_adwords_remarketing_lists/transform.js

```diff
@@ -220,9 +226,62 @@ const processEvent = async (metadata, message, destination) => {
     }
 
     Object.values(createdPayload).forEach((data) => {
-      response.push(responseBuilder(metadata, data, destination));
+      responses.push(responseBuilder(metadata, data, destination));
```

Comment:
Can we do something like this ?
```javascript
const responses = Object.values(createdPayload).forEach(data => {
      return responseBuilder(metadata, data, destination);
});
```
Note: Remove `const responses = [];` at line: 209